examples/login-oidc/client_config_idtoken.js

```diff
@@ -0,0 +1,38 @@
+  // Don't commit this file to your public repos. This config is for first-run
+  //
+
+  exports.creds = {
+    // This is your app's 'REPLY URL' in AAD
+    returnURL: 'http://localhost:3000/auth/openid/return',
+
+    // replace <your_tenant_name> with your tenant name,
+    // which is something like: "*.onmicrosoft.com"
+    identityMetadata: 'https://login.microsoftonline.com/<your_tenant_name>/.well-known/openid-configuration',
+       
+    // This is your app's 'CLIENT ID' in AAD
+    clientID: '2abf3a52-7d86-460b-a1ef-77dc43de8aad',
+    
+    // This is your app's 'key' in AAD. Required if you are doing 'id_token code' or 'code',
```

Comment:
helps if you add `Required if, in the responseType, you are doing...` This has confused me in other examples in the past